sstables/sstables.cc

```diff
@@ -2320,6 +2320,7 @@ static future<bool> do_validate_compressed(input_stream<char>& stream, const sst
     uint64_t offset = 0;
     uint32_t actual_full_checksum = ChecksumType::init_checksum();
 
+    size_t i = 0;
```

Comment:
nitpick: do we really need to maintain yet another counter, i? Can't we calculate it from it-c.offsets.begin() or something?

Reply:
Yes we need the the separate counter because the iterator is a forward iterator.